lib/widgets/creators_corner/exercise_library.dart

```diff
@@ -0,0 +1,222 @@
+import 'package:flutter/material.dart';
+import 'package:workoutpersonalizer_frontend/constants/styles.dart';
+import 'package:workoutpersonalizer_frontend/models/exercise.dart';
+import 'package:workoutpersonalizer_frontend/models/pair.dart';
+import 'package:workoutpersonalizer_frontend/widgets/exercise_card.dart';
+import 'package:workoutpersonalizer_frontend/widgets/creators_corner/filter_list.dart';
+
+class ExerciseLibrary extends StatefulWidget {
+  @override
+  ExerciseLibraryState createState() => ExerciseLibraryState();
+}
+
+class ExerciseLibraryState extends State<ExerciseLibrary> {
+  List<Exercise> allExercises = [];
+  List<Exercise> savedExercises = [];
+  List<String> allExercisesTagList = [];
+  List<String> savedExercisesTagList = [];
+
+  bool allPressed = true;
+  bool savedPressed = false;
+
+  bool sideBarOpened = false;
+
+  @override
+  void initState() {
+    super.initState();
+    String dummyThumbnailUrl = "https://cdn.centr.com/content/17000/16775/images/landscapewidemobile3x-bobby-push-up-16-9.jpg";
+    String dummyVideoUrl = "https://flutter.github.io/assets-for-api-docs/assets/videos/bee.mp4";
+    allExercises = [
+      Exercise(1, "Crunch", "Exercise1 Description", "https://teamworkoutplatform.s3.us-west-2.amazonaws.com/Exercise+1+Thumbnail.png", "https://teamworkoutplatform.s3.us-west-2.amazonaws.com/Exercise+1+.mp4", 1, ["abs", "legs"]),
```

Comment:
I've added the dummy data into the Slack!

Reply:
^ I didn't include the saved but we can add if its needed, lmk the format if so!